src/rewriters/Rewriter.ts

```diff
@@ -48,11 +61,11 @@ abstract class Rewriter {
     return true;
   }
 
-  public rewriteQuery(nodeAndVarDefs: NodeAndVarDefs): NodeAndVarDefs {
+  public rewriteQuery(nodeAndVarDefs: NodeAndVarDefs, _?: Variables): NodeAndVarDefs {
```

Comment:
nit: would prefer to use an variable name here instead of `_` even though it's not being used, since it's an inherited method that will show up in autocomplete